### PR TITLE
Added C6 years to Corvette Grand Sport model

### PR DESCRIPTION
**Pull request by edfooliu**
bit of a drive-by/toe-dipping test

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -1726,6 +1726,10 @@ const allSoloCars = {
       '2013': ['bs', 'sst', 'ssr', 'ssp', 'cams', 'xp', 'xa', 'ssm'],
     },
     'Corvette Grand Sport': {
+      '2010': ['as', 'sst', 'ssr', 'ssp', 'cams', 'xp', 'ssm'],
```
Comment (Bjorn248) [inline src/common.js:1729]: It's probably eligible for XA as well right?

**Review by Bjorn248 — APPROVED** [review]
Looks good to me, this also made me realize that classing overall for the C6 has changed.

The C6, non-ZR1 should go in AS and the base and Z51 C6 goes in BS. I'll try to fix that as part of https://github.com/Bjorn248/scca_classifier/issues/262

File: src/common.js (hunk 1 above)
Comment (edfooliu) [reply src/common.js:1729]: [Latest XStreet rules](https://cdn.connectsites.net/user_files/scca/downloads/000/065/394/2023_CAM_and_Xtreme_Rules_v13.pdf?1677183964&fbclid=IwAR34ifOgFAmvNO_gBy1k1Q1vs5zULefBVm7tKpj6teHt060zy8bbrsuneF8) exclude CAM-eligible cars from XA and XB... 
image

it *should* be XSport legal, but I think the XS category as a whole needs a going-over with how volatile it's been the last couple months.

Comment (Bjorn248) [reply src/common.js:1729]: Ahhhh! I hadn't read the updated rules, then I'll have to go through and remove XA from all the CAM cars now 💀

**Pull request merged**